### PR TITLE
[#1454] Added generic contact form with email and API registration

### PR DESCRIPTION
**Pull request by Bartvaderkin**
If you want to test this enter an email-address the `register_email` field on the OpenKlant config singleton AND add one or more ContactFormSubject's via the inline admins on the same config singleton, then the form should show up on `/contactformulier` to test the email flow.

### CODE REVIEW

**Review by vaszig — CHANGES_REQUESTED** [review]
Let's add a link to the footer as well.

File: src/open_inwoner/openklant/wrap.py [hunk 1]
```diff
@@ -83,10 +91,22 @@ def _fetch_klanten_for_bsn(user_bsn: str, *, client=None) -> List[Klant]:
     return klanten
 
 
+def fetch_klant_for_bsn(user_bsn: str) -> Optional[Klant]:
+    # this is technically a search operation and could return multiple records
+    klanten = _fetch_klanten_for_bsn(user_bsn)
+    if klanten:
+        # let's use the first one
```
[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/openklant/wrap.py:98]: It is very unlikely there would be multiple but the API is implemented as a search so we need to deal with it. 

Yes it is bad API but I checked with Alex.

File: src/open_inwoner/openklant/tests/test_contactform.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        config.klanten_service = None
+        config.contactmomenten_service = None
+        config.register_email = ""
+        config.register_contact_moment = False
+        config.register_bronorganisatie_rsin = ""
+        config.register_type = ""
+        config.register_employee_id = ""
+        config.save()
+
+    # def setupAPIRegister(self, config):
```
[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/openklant/tests/test_contactform.py:38]: nah, just lingering cruft. ill remove it.

File: src/open_inwoner/components/templates/components/Contact/ContactForm.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% load i18n form_tags %}
+
+<div class="contactform__content">
+    {% if has_form_configuration %}
+        {% form form_object=form_object method="POST" id="contact-form" submit_text=_('Verzenden') %}
```
[the remark this answers is not in the document]
Comment (Bartvaderkin) [reply src/open_inwoner/components/templates/components/Contact/ContactForm.html]: Good catch, I updated it to `contactmoment-form`

**Comment by Bartvaderkin** [comment]
I added it to the footer and did some small fixes.

**Review by vaszig — APPROVED** [review]

**Review by alextreme — APPROVED** [review]
Voor nu prima. Zat wel te denken dat het wellicht handiger was geweest als CMS plugin in combinatie met een algemene content CMS pagetemplate, maar gezien de complexiteit van de OpenKlant integratie moeten we dit stap voor stap gaan doen.

**Pull request merged**